src/Model/Contact.php

```diff
@@ -30,6 +30,19 @@
  */
 class Contact extends BaseObject
 {
+	/**
+	 * @name Contact_is
+	 *
+	 * Relationship types
+	 * @{
+	 */
+	const CONTACT_IS_FOLLOWER = 1;
```

Comment:
Please rename to just `FOLLOWER`.

Reply:
Hmm, I would prefer nice prefixes. Like `User::USER_STATUS_LOCKED` or so.

Reply:
Okay, changed my mind.

Reply:
3 minutes, new world record!

More seriously, you already have the class name prefix, and it's always going to be there, so the double notation isn't necessary.

Reply:
And I would also like to do it with the `NETWORK_*` constants. Any suggestions? There is no `Network` class where they could nicely reside.

And same with `PAGE_*`.

Reply:
`NETWORK_*` constants already have been moved to `Friendica\Core\Protocol`, although the legacy constants have been kept for backward compatibility.

`PAGE_*` constants should be moved to `Friendica\Model\Profile`.

Reply:
Okay, new PR for this.

Reply:
And how about `ACCOUNT_TYPE_*`? Also to same class?